src/Repository/FormSubmissionRepository.php

```diff
@@ -74,6 +97,11 @@ public function findFormSubmissions(?string $formInstance = null): array
         return $qb->getQuery()->getArrayResult();
     }
 
+    public function persist(FormSubmission $formSubmission): void
+    {
+        $this->_em->persist($formSubmission);
+    }
+
     public function save(FormSubmission $formSubmission): void
```

Comment:
Maybe deprecate it?

Reply:
Could you explain why please? Is Doctrine's persist outdated?